cmd/webhook/main.go

```diff
@@ -140,6 +142,8 @@ func main() {
 		v1alpha1.SchemeGroupVersion.WithKind("JobSource"):     &v1alpha1.JobSource{},
 		v1alpha1.SchemeGroupVersion.WithKind("CronJobSource"): &v1alpha1.CronJobSource{},
 		v1alpha1.SchemeGroupVersion.WithKind("ServiceSource"): &v1alpha1.ServiceSource{},
+		corev1.SchemeGroupVersion.WithKind("Pod"):             &v1alpha1.SourcePod{},
+		corev1.SchemeGroupVersion.WithKind("Binding"):         &v1alpha1.NOPBinding{},
```

Comment:
A little comment on why you are doing this would be helpful to understand later. (for me 😆)